client/client-multi/client-c/src/kaa_context.h

```diff
@@ -28,22 +28,29 @@
 extern "C" {
 #endif
 
-typedef struct kaa_status_t             kaa_status_t;
+#include "utilities/kaa_log.h"
+#include "kaa_status.h"
+#include "kaa_event.h"
+#include "kaa_profile.h"
+#include "kaa_logging.h"
+#include "kaa_user.h"
```

Comment:
I don't like what happened here. Unless there is an unavoidable need to actually include all of these headers into kaa_context.h (please explain why), let us use forward declarations instead, as it was done previously.